libbeat/docs/shared-template-load.asciidoc

```diff
@@ -82,3 +116,12 @@ at the newest documents. Use this command:
 ----------------------------------------------------------------------
 curl -XDELETE 'http://localhost:9200/{beatname_lc}-*'
 ----------------------------------------------------------------------
+
+[[passing-credentials-template-loading]]
+==== Pass credentials
+
+//REVIEWERS: How do you do this when you are using Winlogbeat and starting as a service? Do you need to configure username/password in the config file? I haven't mentioned the config options in the GS because I think it's better for users not to store the passwords in config files. Is that a correct assumption?
```

Comment:
If you start the Beat as a service, the only option is to add the username/password in config file.  I think we should mention both options in the docs: by changing the config file or in the command line. I wouldn't say one option is better than the other, as when passing username/password in the config file, the user can see the command in history.